tools/c7n_gcp/tests/data/flights/gke-cluster-delete/get-v1-projects-cloud-custodian-locations-us-east1-b-serverConfig_1.json

```diff
@@ -0,0 +1,640 @@
+{
+    "headers": {
+      "content-type": "application/json; charset=UTF-8",
+      "vary": "Origin, X-Origin, Referer",
+      "date": "Wed, 23 Aug 2023 23:04:28 GMT",
```

Comment:
these files can be removed, now that its a filter instead of augment, the additional test files in unrelated tests are now superfluous.